dwoo/Dwoo/Compiler.php

```diff
@@ -1820,7 +1820,7 @@ protected function parseFunction($in, $from, $to, $parsingParams = false, $curBl
 				if ($func === 'tif') {
 					$params[] = $tokens;
 				}
-				$output = call_user_func_array($funcCompiler, $params);
+				$output = call_user_func_array($funcCompiler, array_values($params));
```

Comment:
fix: PHP Fatal error:  Uncaught Error: Unknown named parameter $* in /usr/share/ganglia/dwoo/Dwoo/Compiler.php:1823

Not sure if this would introduce a logic bug, but the web page looks normal.